dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -711,3 +707,53 @@ def test_profile_merge_no_bin_overlap(self):
                                     'Profiles have no overlapping bin methods '
                                     'and therefore cannot be added together.'):
             profiler1 + profiler2
+
+    def test_profile_merge_with_different_options(self):
+        # Creating first profiler with default options
+        options = FloatOptions()
+        options.max.is_enabled = False
+        options.min.is_enabled = False
+
+        data = [2, 4, 6, 8]
+        df = pd.Series(data).apply(str)
+        profiler1 = FloatColumn("Float", options=options)
+        profiler1.update(df)
+
+        # Creating second profiler with separate options
+        options = FloatOptions()
+        options.min.is_enabled = False
+        options.precision.is_enabled = False
+        data2 = [10, 15]
+        df2 = pd.Series(data2).apply(str)
+        profiler2 = FloatColumn("Float", options=options)
+        profiler2.update(df2)
+
+        # Asserting warning when adding 2 profilers with different options
+        with warnings.catch_warnings(record=True) as w:
+            profiler3 = profiler1 + profiler2
+            list_of_warning_messages = []
+            for warning in w:
+                list_of_warning_messages.append(str(warning.message))
+            
+            warning1 = "precision is disabled because it is not enabled in both" \
+                       " profiles."
+            warning2 = "max is disabled because it is not enabled in both " \
+                       "profiles."
+            self.assertTrue(warning1 in list_of_warning_messages)
+            self.assertTrue(warning2 in list_of_warning_messages)
```

Comment:
assertIn as opposed to assertTrue will provide more insight into failures if they occur